src/audio/code_8019AF00.c

```diff
@@ -4211,46 +4265,246 @@ void AudioSfx_LowerSfxSettingsReverb(Vec3f* pos, s8 isReverbLowered) {
     }
 }
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/code_8019AF00/func_8019F5AC.s")
+f32 Audio_SetSyncedSfxFreqAndVolume(f32 freqVolParam) {
+    f32 ret = 1.0f;
+
+    if (freqVolParam > 6.0f) {
+        sSfxSyncedVolume = 1.0f;
+        sSfxSyncedFreq = 1.1f;
+    } else {
+        ret = freqVolParam / 6.0f;
+        sSfxSyncedVolume = (ret * 0.22500002f) + 0.775f;
```

Comment:
```suggestion
        sSfxSyncedVolume = (ret * ((0.09f * 0.1f) / 0.04f)) + 0.775f;
```
Can't find anything better unfortunately :(

Reply:
Or you know, `sSfxSyncedVolume = ret * (1 - 0.775f) + 0.775f;` seems to work :)